doc-site/components/landing-page-card.tsx

```diff
@@ -50,7 +51,7 @@ function LandingPageCard({ imageUrl, children }: LandingPageCardProps) {
   return (
     <Card disableHoverEffect css={cardStyle}>
       <div css={imageWrapperStyle}>
-        <img src={imageUrl} css={imageStyle} />
+        <img src={imageUrl} css={imageStyle} alt="" />
```

Comment:
```suggestion
        <img src={imageUrl} css={imageStyle} alt="Example landing page card image" />
```
? Or take it as a mandatory prop?